openslides/motions/static/js/motions/site.js

```diff
@@ -285,7 +285,23 @@ angular.module('OpenSlidesApp.motions.site', [
     'gettextCatalog',
     'Editor',
     'Config',
-    function(gettextCatalog, Editor, Config) {
+    'formlyConfig',
+    function(gettextCatalog, Editor, Config, formlyConfig) {
+        formlyConfig.setType({
```

Comment:
I moved the type definition to the site-wide formly-configuration in core/site.js to the other custom formly types.